config.yaml

```diff
@@ -6,3 +6,4 @@ organizations:
 days: 7
 filename: "pRs.json"
 prettyprintfile: "myfile.html"
+release-summary-filename: "release-summary.html"
```

Comment:
[nitpick] Please also ignore this from getting copied during docker image creation and Git commit creation.

Reply:
Added the file to `.gitignore`.